### ISSUE TITLE
A bug in version 1.7.3 (10703)

### ISSUE THREAD

**Issue by iw00d** [jenspfahl/EverydayTasks#86]
I'm not entirely sure if this is a bug or a feature request. 

_What I did:_
I just let a schedule that is due every fortnight run for about three weeks before I got around to it and marked it as finished.

_What I expected to happen:_
The task got marked as finished and will show up the next time the two weeks are up.

_What happened instead:_
The first instance, the skipped reminder or most overdue version, got labeled as finished. The second instance lies untouched, asking for feedback of how I performed the task.
The task will show up the next time the two weeks are up.

**Comment by jenspfahl** [comment]
Are you referring to the calendar feature? 
Is it a flexible or fixed schedule?

**Comment by iw00d** [comment]
This is for a fixed task I have set to be reminded every other Thursday.

**Comment by jenspfahl** [comment]
Yes, for a fixed task you have to mark every single instance. Or you fast forward the next due date to a defined date (Skip due date / Custom ...)
